student-work/2022/Rachel-Goldstuck/ReadMe.md

```diff
@@ -1 +1,51 @@
+# **RACHEL GOLDSTUCK**
 
+*TECHNICAL COMMUNICATOR*
+
+Seeking to join an established team of TechComm professionals at a software company
+
+# SUMMARY
+
+An ambitious individual with a passion for the written word for a variety of texts, Rachel is loves to simplify language to assist the reader in their knowledge and understanding.
+
+Rachel spent many years in the educational publishing industry in South Africa. Rachel oversaw the commissioning of multiple educational projects to completion and approval by various Ministries of Education. This included stakeholder and financial management.
+
+Rachel emigrated to Israel in Novemeber 2021 and decided to retrain as a technical communicator.
+
+# TECHNICAL PROFICIENCIES
+
+- Comprehension of HTML and CSS
+- Proficient with Adobe Acrobat Pro
+- Understanding of Madcap Flare
+
+# WORK EXPERIENCE
+
+## PUBLISHING PROFESSIONAL
+
+### Independent | June 2016 – current
+
+- Project management of various texts from concept to publication.
+- Editing, proofreading, materials development, and audits of educational texts in the Southern African Educational Publishing Sector.
+
+## PUBLISHER & DIGITAL PUBLISHER
+
+### Macmillan South Africa | February 2007 – May 2016
+
+- Developing, commissioning and project managing authors and stakeholders in the development of Southern African textbooks for Grade 1-12.
+- Successfully achieved the target of having multiple textbooks listed on the National Catalogue.
+  Project management of the conversion of all textbooks in the Educational Catalogue into digital formats for different portals of learning.
```

Comment:
```suggestion
  - Project management of the conversion of all textbooks in the Educational Catalogue into digital formats for different portals of learning.
```

Reply:
If this is meant to be indented as additional layered content, ignore the previous comment.